allennlp/data/dataset_readers/conll2003.py

```diff
@@ -61,24 +65,23 @@ class Conll2003DatasetReader(DatasetReader):
         Each will have its own namespace: ``pos_tags``, ``chunk_tags``, ``ner_tags``.
         If you want to use one of the tags as a `feature` in your model, it should be
         specified here.
-    coding_scheme: ``str``, optional (default=``IOB1``)
+    convert_to_coding_scheme: ``str``, optional (default=``None``)
         Specifies the coding scheme for ``ner_labels`` and ``chunk_labels``.
-        Valid options are ``IOB1`` and ``BIOUL``.  The ``IOB1`` default maintains
-        the original IOB1 scheme in the CoNLL 2003 NER data.
-        In the IOB1 scheme, I is a token inside a span, O is a token outside
-        a span and B is the beginning of span immediately following another
-        span of the same type.
+        A valid option is ``BIOUL``.
```

Comment:
I think you can just remove this line

Reply:
You mean the 'A valid option is BIOUL' line?

Reply:
yup!